packages/@ngtools/webpack/src/plugin.ts

```diff
@@ -390,9 +390,9 @@ export class AotPlugin implements Tapable {
       // Wait for the plugin to be done when requesting `.ts` files directly (entry points), or
       // when the issuer is a `.ts` file.
       compiler.resolvers.normal.plugin('before-resolve', (request: any, cb: () => void) => {
-        if (request.request.endsWith('.ts')
+        if (this.done && request.request.endsWith('.ts')
```

Comment:
bad logic? shouldn't it be:
```
        if (this.done
            && (request.request.endsWith('.ts')
                || (request.context.issuer && request.context.issuer.endsWith('.ts')))) {
```

Reply:
Ah. I missed the second line (not a fan of the line break here). Fixed in next version.